docs/external-oasis-repos.md

```diff
@@ -30,31 +30,43 @@ Requests to Host a Repository on OASIS
           - http://hcc-cvmfs-repo.unl.edu:8000/cvmfs/dune.osgstorage.org/
         ...
      
-    When PR is approved, check on the `oasis.opensciencegrid.org` host whether the new repository was successfuly signed.
-    There should be sign of it in the log file `/var/log/oasis/generate_whitelists.log`:
+    When the PR is approved, check on the `oasis.opensciencegrid.org` host whether the new repository was successfuly signed.
+    There should be message about it in the log file `/var/log/oasis/generate_whitelists.log`:
 
     ```
     Tue Sep 25 17:34:02 2018 Running add_osg_repository http://hcc-cvmfs-repo.unl.edu:8000/cvmfs/dune.osgstorage.org
 dune.osgstorage.org: Signing 7 day whitelist with masterkeycard... done
     ```
 
-1.  If the respository ends in a new domain name `domain.name` that has not been distributed before, then place a copy of the
-    `domain.name.pub` public key into `/srv/etc/keys` on both `oasis-replica` and
-    `oasis-replica-itb`. If you do not have that key, then ask the repository service representative how to obtain it. In
-    order to support CVMFS client versions 2.2.X, also make a symbolic link of `domain.name.conf`
-    `/cvmfs/config-osg.opensciencegrid.org/etc/cvmfs/domain.d` pointing to `default.conf`. This symbolic link has to be
-    created on the `oasis-itb` machine's copy of the `config-osg.opensciencegrid.org` repository and then copied to
-    production with the `copy_config_osg` command on the oasis machine.
+1.  If the respository ends in a new domain name that has not been distributed before, 
+    a new domain key will be needed on oasis-replica which should get automatically downloaded from github.
```

Comment:
Could we perhaps link the exact github so it's clear where to look in case of problem with download for example? Like we link few lines below, for example.

Reply:
It's the same place.  I'll just move up the link.